src/api/app/views/webui2/webui/kiwi/images/_build_status.html.haml

```diff
@@ -0,0 +1,26 @@
+- @project = @image.package.project
+- @build_results.each_pair do |package, results|
+  %ul.list-group.list-group-flush#package-buildstatus
+    - results.each do |result|
+      %li.list-group-item
+        - repository = @project.repositories.find_by_name(result.repository)
+        - if !(repository && repository.architectures.pluck(:name).include?(result.architecture))
+          %i.fas.fa-clock.text-warning{ title: 'This result is outdated' }
+        - else
+          :ruby
+            icon = webui2_repo_status_icon(result.state)
+            description = webui2_repo_status_description(result.state, result.details)
+          %i{ class: icon, data: { content: description, placement: 'top', toggle: 'popover' } }
+        = result.architecture
+        = webui2_arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
+
+.card-body
+  = link_to(package_binaries_path(project: @project, package: @image.package, repository: 'images'), title: 'Download images') do
+    %i.text-info.fa.fa-download
+    Download images
+= javascript_tag do
+  :plain
+    $('.unresolvable, .blocked').click(function() {
+      var title = $(this).attr('title');
+      alert(title);
```

Comment:
We use [bootstrap's popovers](https://getbootstrap.com/docs/4.1/components/popovers/) for the build results on the package show views. I would use them here as well to stay consistent.

Reply:
Yeah, that code is not needed, helper should take care of providing proper HTML for popovers in that place